__tests__/app.test.js

```diff
@@ -113,7 +116,41 @@ describe("GET /api/articles", () => {
       .get("/api/kiwi")
       .expect(404)
       .then(({ body }) => {
-        expect(body.msg).toBe("Endpoint not found");
+        expect(body.msg).toBe("Route not found");
+      });
+  });
+});
+
+//task 6
+describe("GET /api/articles/:article_id/comments", () => {
+  const validArticleIds = [1, 3, 5, 6, 9];
+  validArticleIds.forEach((article_id) => {
+    test("200: responds with an array of the comments for specific article id", () => {
+      return request(app)
+        .get(`/api/articles/${article_id}/comments`)
+        .expect(200)
+        .then(({ body: { comments } }) => {
+          comments.forEach((comment) => {
```

Comment:
remember when using a foreach we'd need to assert the length of the array to ensure we dont get any false passes